app/models/manageiq/providers/ibm_power_hmc/infra_manager/host.rb

```diff
@@ -1,6 +1,42 @@
 class ManageIQ::Providers::IbmPowerHmc::InfraManager::Host < ::Host
   include ManageIQ::Providers::IbmPowerHmc::InfraManager::MetricsCaptureMixin
 
+  supports :stop
+  supports :shutdown
+
+  def shutdown
+    $ibm_power_hmc_log.info("#{self.class}##{__method__}")
+    ext_management_system.with_provider_connection do |connection|
+      connection.poweroff_managed_system(ems_ref)
+    rescue IbmPowerHmc::Connection::HttpError => e
+      $ibm_power_hmc_log.error("error powering off LPAR #{ems_ref} with params=#{params}: #{e}")
+      raise
+    end
+
+  end
+
+  def start
+    $ibm_power_hmc_log.info("#{self.class}##{__method__}")
+    ext_management_system.with_provider_connection do |connection|
+      connection.poweron_managed_system(ems_ref,
+        :params   => {"operation" => on}
```

Comment:
Really minor but if you're not aligning the hash to anything best to leave out the extra padding
```suggestion
        :params => {"operation" => on}
```